BuildSimHubAPI/helpers/parametric_model.py

```diff
@@ -12,9 +12,9 @@ class ParametricModel:
     BASE_URL = 'https://my.buildsim.io/'
 
     def __init__(self, userKey, model_key, base_url=None):
```

Comment:
Hi @weilix88, This looks great. You're missing `userKey` input here which should be `user_key`.

Reply:
will fix that, thanks!